react/components/GalleryItem.js

```diff
@@ -22,21 +22,17 @@ export default class GalleryItem extends Component {
     if (!product) {
       return null
     }
+    const compareSKUs = (skuA, skuB) => path(['sellers', '0', 'commertialOffer', 'AvailableQuantity'], skuB) - path(['sellers', '0', 'commertialOffer', 'AvailableQuantity'], skuA)
```

Comment:
Extract the `path(['sellers', '0', 'commertialOffer', 'AvailableQuantity']` into a function and call it

Reply:
And use [comparator](https://ramdajs.com/docs/#comparator) with [subtract](
https://ramdajs.com/docs/#subtract)

Reply:
Avoid naming parameters when you can omit them. Ramda uses curry for every function they export, as I've pointed in the last PR

Reply:
And make it clean in the name of the comparator which criteria is being used.